tests/queries/0_stateless/02475_date_time_schema_inference_bug.sql

```diff
@@ -0,0 +1 @@
+select * from format(CSV, '""');
```

Comment:
Wouldn't it be better to add the test query to `tests/queries/0_stateless/02456_datetime_schema_inference.sql` ?